hoomd/md/pair/aniso.py

```diff
@@ -47,6 +47,10 @@ class AnisotropicPair(Pair):
         mode (`str`, optional) : the energy shifting mode, defaults to "none".
     """
 
+    # Module where the C++ class is defined. Reassign this when developing an
+    # external plugin.
+    _ext_module = _md
```

Comment:
We have a plugin class that successfully inherits this class and overrides `_ext_module`:

https://github.com/mphowardlab/azplugins/blob/0c520e05afe59ef07ab3405fccaff4d90524fc45/src/pair.py#L371

I think this change is not needed because `_ext_module` is inherited from `Pair`.

Reply:
Ah! Great. I'll remove it.